geometry/convex_hull.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include <vector>
+
+namespace m2
+{
+// Builds a convex hull around |points|.
```

Comment:
The hull polygon does not have three points lying on the same straight line.
The hull polygon points are listed in the order of a counterclockwise traversal.

Reply:
Done